docker/docker-compose.yaml

```
@@ -113,7 +113,7 @@ services:
     restart: on-failure
 
   sensing-perception:
-    image: ghcr.io/autowarefoundation/autoware:universe-sensing-perception
+    image: ${PERCEPTION_IMAGE}
```

```suggestion
    image: ${SENSING_PERCEPTION_IMAGE}
```
is more descriptive?

---

fixed  [`0ed3983` (#6373)](https://github.com/autowarefoundation/autoware/pullcommits/0ed3983998be3b035ff6e75a995c2ef24d2647c9)